### DISCUSSION TITLE
Error occurred when executing DWPreprocessor:

### DISCUSSION THREAD

**Discussion by mendo878** [community/community#133597] [Programming Help]
### Select Topic Area

Bug

### Body

help me pls

Error occurred when executing DWPreprocessor:

cannot import name 'packaging' from 'pkg_resources' (C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\venv\lib\site-packages\pkg_resources\__init__.py)

File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\node_wrappers\dwpose.py", line 87, in estimate_pose
out = common_annotator_call(func, image, include_hand=detect_hand, include_face=detect_face, include_body=detect_body, image_and_json=True, resolution=resolution)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\utils.py", line 85, in common_annotator_call
np_result = model(np_image, output_type="np", detect_resolution=detect_resolution, **kwargs)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\node_wrappers\dwpose.py", line 83, in func
pose_img, openpose_dict = model(image, **kwargs)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\dwpose\__init__.py", line 262, in __call__
poses = self.detect_poses(input_image)
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\dwpose\__init__.py", line 252, in detect_poses
keypoints_info = self.dw_pose_estimation(oriImg.copy())
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\dwpose\wholebody.py", line 80, in __call__
import torch.utils.benchmark.utils.timer as torch_timer
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\venv\lib\site-packages\torch\utils\benchmark\__init__.py", line 2, in
from torch.utils.benchmark.utils.timer import * # noqa: F403
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\venv\lib\site-packages\torch\utils\benchmark\utils\timer.py", line 8, in
from torch.utils.benchmark.utils import common, cpp_jit
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\venv\lib\site-packages\torch\utils\benchmark\utils\cpp_jit.py", line 13, in
from torch.utils import cpp_extension
File "C:\Users\ganic\Desktop\Sytability Matrix\Data\Packages\ComfyUI\venv\lib\site-packages\torch\utils\cpp_extension.py", line 28, in
from pkg_resources import packaging # type: ignore[attr-defined]

**Comment by github-actions** [comment]
**💬 Your Product Feedback Has Been Submitted 🎉**

Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

**Here's what you can expect moving forward ⏩**
- Your input will be carefully reviewed and cataloged by members of our product teams. 
  - Due to the high volume of submissions, we may not always be able to provide individual responses.
  - Rest assured, your feedback will help chart our course for product improvements.
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 
  - We may 'Answer' your discussion if there is a current solution, workaround, or roadmap/changelog post related to the feedback.

**Where to look to see what's shipping 👀**
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

**What you can do in the meantime 💻**
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by metagamma** [comment]
Hello, @mendo878. Try updating setuptools and pkg_resources:
pip install --upgrade setuptools pkg_resources
If that doesn't work, try uninstalling and reinstalling setuptools:
pip uninstall setuptools
pip install setuptools

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mendo878 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
